playbooks/installer/ospd/overcloud/templates/overcloud_deploy.sh.j2

```diff
@@ -8,7 +8,7 @@ openstack overcloud deploy --debug \
                        --neutron-network-type {{ installer.overcloud.network.backend }} \
 {% if installer.overcloud.network.backend == "vlan" %}
                        --neutron-disable-tunneling \
-                       --neutron-bridge-mappings {{ installer.undercloud.config.physical_network }}:br-ex,tenant:br-isolated \
+                       --neutron-bridge-mappings {{ installer.undercloud.network.physical_network }}:br-ex,tenant:br-isolated \
```

Comment:
I've changed the settings/installer/ospd.yml structure. Now all the undercloud network parameters are stored under installer.undercloud.network.

@tkammer  let me know if that is acceptable, or we need to go back to 'installer.undercloud.config'